api/src/main/resources/workbench-api.yaml

```diff
@@ -2320,23 +2320,17 @@ paths:
       tags:
       - institution
       description: 'Gets the list of all Institutions. Requires INSTITUTION_ADMIN
-        authority.
```

Comment:
There's a bit of "clean up the awkward single-quote whitespace" here. We should probably do this for the whole file at some point, but I couldn't help myself from fixing it for institution APIs while I was here.